server/pulp/server/agent/context.py

```diff
@@ -14,7 +14,8 @@
 from gofer.messaging import Queue
 
 from pulp.server.agent.auth import Authenticator
-from pulp.server.agent.direct.services import Services, ReplyHandler
+from pulp.server.agent.connector import get_url, add_connector
```

Comment:
Consider alphabetizing imports from left to right. ie: swap get_url and add_connector